jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcRepository.java

```diff
@@ -309,6 +309,9 @@ protected Condition getConditionOnField(
                 throw new InvalidQueryFiltersException("Label field value must but instance of Map");
             }
         }
+        if(field==QueryFilter.Field.KIND){
```

Comment:
```suggestion
        if (field == QueryFilter.Field.KIND) {
```

Nitpick but as you start contributing regularly would be great if you could follow our coding format ;)

Reply:
done ;)